ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -0,0 +1,383 @@
+"""
+This module deploys the openshift-logging on the cluster
+EFK stack
+"""
+
+import logging
+import json
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_obj
+from ocs_ci.utility import templating
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.retry import retry
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+def create_namespace(yaml_file):
+    """exec_cmd_on_pod(self, command, out_yaml_format=True):
+    Creation of namespace "openshift-operators-redhat"
+    for Elasticsearch-operator and "openshift-logging"
+    for ClusterLogging-operator
+
+    Args:
+        yaml_file (str) : Path to yaml file to create namespace
+
+    Returns:
+        namespace (Boolean): True if namespace is created and active, False
+        otherwise
+
+    Example:
+        create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+    """
+
+    namespaces = ocp.OCP(kind=constants.NAMESPACES)
+
+    logger.info("Creating Namespaces")
+    namespaces.create(yaml_file=yaml_file)
+    namespace = namespaces.get()
+    for i in namespace:
+        if namespace['items'][i]['status']['phase'] == 'Active':
+            logger.info("Creation of namespace "
+                        f"{namespace['items'][i]['metadata']['name']}"
+                        " is successful")
+    return bool(namespace)
+
+
+def create_elasticsearch_operator_group(yaml_file, resource_name):
+    """
+    Creation of operator-group for Elastic-search operator
+
+    Args:
+        yaml_file (str) : Path to yaml file to create operator group for
+                          elastic-search
+        resource_name (str) : Name of the operator group to create for
+                              elastic-search
+
+    Returns:
+        Boolean : True if operator group for elastic search is created
+                  successfully, false otherwise
+
+    Example:
+        create_elasticsearch_operator_group(
+        constants.EO_OG_YAML, 'openshift-operators-redhat'
+        )
+
+    """
+
+    es_operator_group = ocp.OCP(
+        kind=constants.OPERATOR_GROUP, namespace='openshift-operators-redhat'
+    )
+
+    es_operator_group.create(yaml_file=yaml_file)
+    try:
+        es_operator_group.get(resource_name, out_yaml_format=True)
+        logger.info(f'The Operator group is created successfully')
+    except CommandFailed:
+        logger.error(f'The resource is not found')
```

Comment:
No need to use f-strings if we aren't formatting anything, fix for all other instances as well.

```suggestion
        logger.error('The resource is not found')
```